BGL/include/CGAL/boost/graph/copy_face_graph.h

```diff
@@ -208,8 +212,49 @@ void copy_face_graph(const SourceMesh& sm, TargetMesh& tm,
                        sm_vpm, tm_vpm);
 }
 
+template<typename PMAP>
+struct Output_iterator_functor
+{
+  typedef typename boost::property_traits<PMAP>::key_type input_t;
+  typedef typename boost::property_traits<PMAP>::value_type output_t;
+  PMAP map;
+  Output_iterator_functor(PMAP map)
+    :map(map)
+  {
+  }
+  void operator()(const typename std::pair<input_t, output_t>& pair)
+  {
+    put(map, pair.first, pair.second);
+  }
+  
+};
+
+struct Emptyset_iterator_functor
+{
+  Emptyset_iterator_functor()
+  {}
+              
+  template <class T>
+  void operator()(const T&)
+  {}
+  
+};
+
+template<typename PMAP>
+Output_iterator_functor<PMAP> make_functor(PMAP map)
+{
+  return Output_iterator_functor<PMAP>(map);
```

Comment:
why don't you directly call make_function_output_iterator withing this function? This would remove the need for `Emptyset_iterator_functor`